ci/Jenkinsfile

```diff
@@ -170,11 +170,9 @@ def getNewBuildName() {
   /* For URLs we need to parse the filename to get attributes. */
   if (params.BUILD_SOURCE.startsWith('http')) {
     def tokens = utils.parseFilename(utils.baseName(params.BUILD_SOURCE))
-    return [
-      (tokens.tstamp ?: tokens.version), tokens.commit, tokens.build
-    ].grep().join('-')
```

Comment:
Why are your removing the other information?

Reply:
@jakubgs to improve a bit the naming of the run, it is hard to read it now. Maybe lets keep the code but just comment it for now with a note, we will experiment for a while with that. Would that be okay?

Reply:
Well, one thing we can do is make the build `displayName` more brief, but include the whole info about App used in the build description.

Reply:
Also, honestly, the inclusion of hostname is kinda silly in job name. We could add that to description for debugging.

Reply:
Here's a better solution in my opinion:

* https://github.com/status-im/desktop-qa-automation/pull/217

Reply:
reverted